### ISSUE TITLE
perniosis and chilblain lesions

### ISSUE THREAD

**Issue by cmungall** [obophenotype/human-phenotype-ontology#3755]
Does this disease belong in MONDO?

https://rarediseases.org/rare-diseases/perniosis/
https://rarediseases.info.nih.gov/diseases/7373/perniosis

It may be more straightforward to treat as a phenotypic feature. 

How would it relate to:

```
[Term]
id: HP:0009710
name: Chilblain lesions
namespace: human_phenotype
def: "Chilblains are acral ulcers." [HPO:curators]
xref: UMLS:C4024224
is_a: HP:0001167  ! Abnormality of finger
```

I would recommend refining the definition here to follow G-D. "Chilblains are acral ulcers *that* ..."

While the finger is commonly affected it can also affect the feet, e.g. https://www.mayoclinic.org/diseases-conditions/chilblains/symptoms-causes/syc-20351097

currently we have two annotations to this phenotype, two forms of AGS, one IEA one TAS:
https://hpo.jax.org/app/browse/term/HP:0009710

I believe the lesions are similar to those in chilblain lupus, but we have HP:0200042-Skin ulcer, nothing more precise. Perhaps this is OK as I believe the AGR lesions are exacerbated by cold, thus "true" chilblains, but in CL this is not the case? https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1592563/

For a comparison with other ontologies, see:

https://www.ncbi.nlm.nih.gov/medgen/C4024224 HP+MESH
https://www.ncbi.nlm.nih.gov/medgen/C0008058 SNOMED+NCIT+MESH  (mesh has both chilblains and chilblain lesions)

and:

```
[Term]
id: GARD:0007373
name: Perniosis
def: " Perniosis are itchy and/or tender red or purple bumps that occur as a reaction to cold.  In severe cases, blistering, pustules, scabs and ulceration may also develop. Occasionally, the lesions may be ring-shaped. They may become thickened and persist for months. Perniosis is a form of vasculitis . Signs and symptoms occur hours after cold exposure. Risk factors for perniosis include having poor blood circulation (such as due to diabetes or smoking), a family history of perniosis, poor nutrition, and low body weight. Perniosis may occur alone or in association with an autoimmune condition (e.g., lupus, scleroderma), bone marrow disorder , or cancer. Treatment aims to relieve symptoms and prevent infection. Lifestyle/adaptive changes may also be recommended to prevent future symptoms.[8142]" [https://rarediseases.info.nih.gov/diseases/7373/perniosis]
subset: gard_rare  ! GARD rare disease subset
synonym: "Chilblains" RELATED []
synonym: "Cold-induced painful or pruritic erythematous or violaceous acral papular or nodular lesions" RELATED []
synonym: "Idiopathic perniosis" RELATED []
synonym: "Pernio" RELATED []
xref: UMLS:C0008058
property_value: http://www.w3.org/2000/01/rdf-schema#seeAlso https://rarediseases.info.nih.gov/diseases/7373/perniosis

[Term]
id: MESH:D002647
name: Chilblains
namespace: CTD_disease_ontology
def: "Recurrent localized itching, swelling and painful erythema on the fingers, toes or ears, produced by exposure to cold." [MESH:D002647]
synonym: "Chilblain" RELATED []
synonym: "Idiopathic Pernioses" RELATED []
synonym: "Idiopathic Perniosis" RELATED []
synonym: "Pernio" RELATED []
synonym: "Pernios" RELATED []
synonym: "Pernioses" RELATED []
synonym: "Pernioses, Idiopathic" RELATED []
synonym: "Perniosis" RELATED []
synonym: "Perniosis, Idiopathic" RELATED []
is_a: MESH:D005627  ! Frostbite

[Term]
id: NCIT:C84631
name: Chilblains
def: "A rare condition characterized by skin inflammation, blister formation, swelling and ulcerations in the extremities. It occurs in persons exposed to cold temperatures." []
synonym: "Chilblains" EXACT []
is_a: NCIT:C53543  ! Rare Non-Neoplastic Disorder
property_value: NCIT:NHC0 "C84631" xsd:string
property_value: NCIT:P106 "Disease or Syndrome" xsd:string
property_value: NCIT:P108 "Chilblains" xsd:string
property_value: NCIT:P207 "C0008058" xsd:string

[Term]
id: SCTID:37869000
name: chilblains (disorder)
synonym: "chilblain" RELATED []
synonym: "chilblains" EXACT []
synonym: "erythema pernio" RELATED []
synonym: "pernio" RELATED []
synonym: "perniosis" RELATED []
is_a: SCTID:11925005  ! effects of reduced temperature (disorder)
is_a: SCTID:128605003  ! disorder of extremity (disorder)
is_a: SCTID:234014009  ! local cold injury (disorder)
```

**Comment by pnrobinson** [comment]
New definition:
Chilblains, also called perniosis, are an inflammatory skin condition related to an abnormal vascular response to the cold. We are unaware of a reliable estimate of incidence. It typically presents as tender, pruritic red or bluish lesions located symmetrically on the dorsal aspect of the fingers, toes, ears and nose. Less commonly, reports describe involvement of the thighs and buttocks.1 The lesions present hours after exposure to cold and usually resolve spontaneously in one to three weeks.

https://www.ncbi.nlm.nih.gov/pubmed/18325315
https://www.ncbi.nlm.nih.gov/pubmed/22025653/

=> I am changing the label to Chilblains. These can be a feature of other diseases, (https://hpo.jax.org/app/browse/term/HP:0009710), and so I think it is justified to call them a phenotypic feature. There is some indication that people who get chilblaions may be at increased risk of lupus and other autoinflammatory diseases, and thus one could say that chilblains are a facultative feature of lupus etc.  There probably is also simply isolated chilblains which could be conceived of as a disease with one phenotypic feature

**Comment by pnrobinson** [comment]
ps, please close this issue if you agree--let's remember to make a MONDO HPO annotation for this!

**Comment by cmungall** [comment]
Thank!
